utils/validation.py

```diff
@@ -98,27 +98,27 @@ def validate_tick(tick, previous_tick=None, possible_previous_ticks=None,
 
     if not validate_schema(tick_copy, 'tick_schema.json'):
         logger.debug("Tick failed schema validation")
-        return False
+        return False, False
```

Comment:
Cannot use this construction, it breaks timeminer.py here: 

if not validate_tick(tick, prev_tick, possible_previous, verbose=False):

_(breaks by doing if not (Bool, Bool) which has unintended consequences)_

The validate_XYZ functions all only return 1 Boolean, let's keep it that way otherwise the usage will be very confusing

Reply:
Alright but then I gotta split up the function which will also be ugly.